protocol/triple/triple_protocol/handler_stream_compat.go

```diff
@@ -92,6 +92,10 @@ func generateCompatStreamHandlerFunc(
 	interceptor Interceptor,
 ) StreamingHandlerFunc {
 	implementation := func(ctx context.Context, conn StreamingHandlerConn) error {
+		ctx = metadata.NewIncomingContext(ctx, metadata.MD(conn.ExportableHeader()))
+		// staticcheck error: SA1029. Stub code generated by protoc-gen-go-triple makes use of "XXX_TRIPLE_GO_INTERFACE_NAME" directly
+		//nolint:staticcheck
+		ctx = context.WithValue(ctx, "XXX_TRIPLE_GO_INTERFACE_NAME", procedure)
```

Comment:
Code format suggestion: Can this **XXX_TRIPLE_GO_INTERFACE_NAME** be defined in the common/constants directory?